File: SpringBootServer/src/main/java/com/missile/websocket/bean/MissileMessage.java
package com.missile.websocket.bean;

/**
 * Created by Mario on 2017/11/23 0023.
 * WS消息载体
 */
@SuppressWarnings("unused")
public class MissileMessage {
    private int missileType;
    private String missileText;
    private String missileImgUrl;
    private String avatar;
    private String nickName;

    public String getAvatar() {
        return avatar;
    }

    public void setAvatar(String avatar) {
        this.avatar = avatar;
    }

    public String getNickName() {
        return nickName;
    }

    public void setNickName(String nickName) {
        this.nickName = nickName;
    }

    public int getMissileType() {
        return missileType;
    }

    public void setMissileType(int missileType) {
        this.missileType = missileType;
    }

    public String getMissileText() {
        return missileText;
    }

    public void setMissileText(String missileText) {
        this.missileText = missileText;
    }

    public String getMissileImgUrl() {
        return missileImgUrl;
    }

    public void setMissileImgUrl(String missileImgUrl) {
        this.missileImgUrl = missileImgUrl;
    }
}

File: Client/Missile/Server/MissileMessage.cs
using Missile.Missile;
using System;

namespace Missile
{
    /// <summary>
    /// 交互信息载体
    /// </summary>
    class MissileMessage
    {
        //消息类型
        public MISSILE_TYPE missileType { set; get; }
        //文本数据
        public String missileText { set; get; }
        //图片数据
        public String missileImgUrl { set; get; }
    }
}
